gtsam/discrete/AlgebraicDecisionTree.h

```diff
@@ -28,9 +28,15 @@ namespace gtsam {
    * TODO: consider eliminating this class altogether?
    */
   template<typename L>
-  class AlgebraicDecisionTree: public DecisionTree<L, double> {
+  class GTSAM_EXPORT AlgebraicDecisionTree: public DecisionTree<L, double> {
+    /// Default method used by `formatter` when printing.
+    static std::string DefaultFormatter(const L& x) {
```

Comment:
We can have this and the `valueFormatter` in the DecisionTreeFactor be the default methods?